rsts/community/roadmap.rst

```diff
@@ -6,35 +6,36 @@ Roadmap
+
+Thus, it is extremely important to let the community know about your use cases and how we could adapt parts of Flyte to meet those requirements. We welcome collaboration and contributions, but please follow our Contribution Guidelines.
 
 
 Milestones and Releases
 ========================
-Flyte consists of many components and services. In true Agile fashion each service is independently iterated and co-ordiated by maintaing backwards compatible contracts using protobuf defined in `idl <https://flyte.readthedocs.io/projects/flyteidl/en/latest/>`__. Thus components like flytekit, flytepropeller, datacatalog are independently versioned.
+Flyte consists of many components and services. In true Agile fashion, each service is independently iterated and coordinated by maintaining backwards-compatible contracts using protobuf defined in `FlyteIDL <https://flyte.readthedocs.io/projects/flyteidl/en/latest/>`__. Thus components like lytekit, Flyte Propeller, and Datacatalog are independently versioned.
 
-We have decided to release a new version of the overall platform in the `flyte repo <https://github.com/flyteorg/flyte>`_ every month. Thus we create one milestone for end of every month which points to a new release of
+We have decided to release a new version of the overall platform in the `Flyte repo <https://github.com/flyteorg/flyte>`_ every month. Thus we create one milestone for end of every month which points to a new release of
 Flyte. This may change in the future, but to match our velocity of development this is our preferred option. Every release will be associated with a CHANGELOG (in the repo).
 
+
 Change management
 ------------------
-To ensure that changes are trackable and the history is explainable, we use a slightly cumbersome but helpful process as follows. Some of these are our immediate goals
-- Every PR associated with an issue (automatic searchable documentation)
-- Large PR’s associated with Proposals
-- Every major change is associated with documentation
-- Owner files for all repositories
+To ensure that changes are trackable and the history is explainable, we use a slightly cumbersome but helpful process as follows. Some of these are our immediate goals:
+
+- Every PR is associated with an issue (automatic searchable documentation).
+- Large PR’s are associated with Proposals.
+- Every major change is associated with documentation.
+- Owner files are present for all repositories.
 
 Release Train
 --------------
 - We tag issues with milestones, every new issue will be associated with the next milestone. If the issue is not completed by the milestone, or the contributor feels it may slip the deadline, they should manually move it to the next milestone. Every issue not removed, will be moved to the next milestone.
```

Comment:
```suggestion
- We tag issues with milestones where every new issue is associated with the next milestone. If the issue couldn't be completed before the milestone, or the contributor feels it may slip the deadline, they should manually move it to the next milestone. Every issue not removed will be moved to the next milestone.
```